### PR TITLE
Fix potential race conditions during SDN setup

### PR DESCRIPTION
**Pull request by pravisankar**
closes: https://github.com/openshift/openshift-sdn/issues/145

Depends on: https://github.com/openshift/openshift-sdn/pull/146

### CODE REVIEW

**Comment by pravisankar** [comment]
@rajatchopra @dcbw @danwinship PTAL

**Comment by pravisankar** [comment]
Updated, PTAL

File: plugins/osdn/osdn.go [hunk 1]
```diff
@@ -71,32 +74,27 @@ func (oi *OsdnRegistryInterface) CreateSubnet(nodeName string, sub *osdnapi.Subn
...
 	for {
 		eventType, obj, err := subnetEventQueue.Pop()
 		if err != nil {
 			return err
 		}
+		hs := obj.(*api.HostSubnet)
+		if checkCondition && skipObject(hs.ObjectMeta.ResourceVersion, startVersion) {
+			continue
+		}
+		checkCondition = false
```
Comment (danwinship) [inline plugins/osdn/osdn.go]: Another repeated pattern (everything from "subnetEventQueue, reflector := ..." to "checkCondition = false".

**Comment by pravisankar** [comment]
Updated, repetitive patterns are clubbed into methods.
Travis/Jenkins will fail as it needs openshift-sdn bump pr after merging https://github.com/openshift/openshift-sdn/pull/146
PTAL

**Comment by pravisankar** [comment]
@rajatchopra Added bump commit, please review and merge

**Comment by rajatchopra** [comment]
LGTM [merge]

**Pull request merged**